### ISSUE TITLE
Brew Ninja - Lead Front End Developer - Regina/Canada

### ISSUE THREAD

**Issue by sheam** [HackRegina/jobs#74]
### TLDR
We are looking for a React developer (intermediate to senior) to take over our front-end project.  

## Brew Ninja
Brew Ninja is a new-to-market software (as a solution) package for brewery management. Our product helps breweries run their business more efficiently to ensure that the Craft Brewery Boom continues to grow. 

The Brew Ninja team is growing quickly 🚀 but there are still plenty of opportunities to play a key role in taking our product to the next level, and we are looking for a developer up for that challenge. 

We are big believers in culture and think you will love working with us.

## The Role
We are in need of someone who has experience working on large React projects and has a passion for developing and organizing projects using best practices. As the front end lead you will shape the project going forward to ensure that the front end code base stays in great shape. You will work with start back-end developers to ensure all your API needs are met!

This position is an opportunity to get in on the ground floor, grow your role, and have a genuine impact on the product. 

You will be responsible for mentoring younger developers, as well as reporting directly to management.

The ideal developer will have experience building rich web applications using the React Framework in Typescript. Applicants that also excel in UX design and CSS will be given the highest consideration.

### Opportunity

 - Direct impact on product and team
- Work with motivated, like-minded talent
- Flexible hours including the option of remote work
- Experience the dynamic/fast-pace of a startup

### Qualifications

- At least 2 years of experience building React applications
- Passionate about providing an excellent user experience
- Hold user code quality, best practices, and security at a high standard.

#### Relevant Technologies

- ReactJS
- TypeScript
- Web API experience
- webpack
- Azure/AWS
- Git
- JIRA/VSTS/DevOps

### Location
- Regina, SK
- Remote OK

## Next steps
If you are ready for this challenge, email your CV to jobs at brewninja.net and mention Front-End Developer.

---

## Meta

- [x] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [x] Remote OK
- [ ] Fully Remote
- [ ] Hourly
- [x] Salary
